deploy/configs/frontend_config.py

```diff
@@ -128,13 +145,30 @@ def create_react_env_file(
     region = os.environ.get('deployment_region', 'eu-west-1')
     enable_cw_rum = os.environ.get('enable_cw_rum', 'False')
     reauth_ttl = os.environ.get('reauth_ttl', '5')
```

Comment:
The amount of variables in this block and in the one in `deploy/stacks/pipeline.py` is getting very long. For custom_auth we are reading CodeBuild env variables that are PLAINTEXT. Maybe it is time to review this process, as a proposal we could use PARAMETER_STORE type of CodeBuild env vars: https://docs.aws.amazon.com/codebuild/latest/APIReference/API_EnvironmentVariable.html We would store the cdk.json as an SSM parameter and make it available as environment variables or directly reading it in the frontend_config script. What do you think @noah-paige?

Reply:
Think we should definitely look into using SSM - can pick this up as an enhancement task

Reply:
I added it as a comment in the issue https://github.com/awslabs/aws-dataall/issues/899